src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -3302,7 +3302,36 @@ private void StepRunLoop_Core(bool force = false)
 				UpdateToolsAfter();
 			}
 
+
+			// Mutes the game if EmuHawk is unfocused or minimized
+			// An option in the sound settings needs to be turned on for this to work.
+			if (Config.MuteInBG) {
+				if(WindowState == FormWindowState.Minimized || Form.ActiveForm == null) {
+					Sound.StopSound();
+				}
+				else { 
+					Sound.StartSound();
+				}
+			} else { 
+				Sound.StartSound();
+			}
+
+			// Mutes the game if the current FPS reaches the threshold chosen by the user or lower
+			// An option in the sound settings needs to be turned on for this to work.
+			if(Config.MuteOnLag) {
+				if (_lastFps <= (double)Config.FPSThreshold)
+				{
+					Sound.StopSound();
+				}
+				else
+				{
+					Sound.StartSound();
+				}
+			}

```

Comment:
Assuming this logic is correct, it can be simplified *a lot*.

Reply:
How is that?